gems/pending/MiqVm/MiqVm.rb

```diff
@@ -32,18 +32,23 @@ def initialize(vmCfg, ost=nil)
         # If we're passed a snapshot ID, then obtain the configration of the
         # VM when the snapshot was taken.
         #
+        # TODO: move to MiqVmwareVm
         if (@vim = @ost.miqVim)
             $log.debug "MiqVm::initialize: accessing VM through server: #{@vim.server}" if $log.debug?
             @vimVm = @vim.getVimVm(vmCfg)
             $log.debug "MiqVm::initialize: setting @ost.miqVimVm = #{@vimVm.class}" if $log.debug?
             @ost.miqVimVm = @vimVm
             @vmConfig = VmConfig.new(@vimVm.getCfg(@ost.snapId))
+        # TODO: move this to MiqRhevmVm.
         elsif (@rhevm = @ost.miqRhevm)
             $log.debug "MiqVm::initialize: accessing VM through RHEVM server" if $log.debug?
+            $log.debug "MiqVm::initialize: vmCfg = #{vmCfg}"
             @rhevmVm = @rhevm.get_vm(vmCfg)
             $log.debug "MiqVm::initialize: setting @ost.miqRhevmVm = #{@rhevmVm.class}" if $log.debug?
             @ost.miqRhevmVm = @rhevmVm
-            @vmConfig = VmConfig.new(@rhevmVm.getCfg(@ost.snapId))
+            @vmConfig = VmConfig.new(getCfg(@ost.snapId))
+            $log.debug "MiqVm::initialize: @vmConfig.getHash = #{@vmConfig.getHash.inspect}"
+            $log.debug "MiqVm::initialize: @vmConfig.getDiskFileHash = #{@vmConfig.getDiskFileHash.inspect}"
```

Comment:
@roliveri Is MiqVolumeManager the same as MiqVm?  I see log lines below with the former.  An alternative would be use to use class.name or self.class.name and __method__  so we don't have hardcoded class/method names that should easily change.  It's not a problem, just noticed we're repeating the same log prefix a lot.  We could also use the _log method that @matthewd implemented for the rails app code.